packages/odyssey-storybook/src/components/odyssey-mui/Dialog/Dialog.stories.tsx

```diff
@@ -119,6 +119,18 @@ const storybookMeta: Meta<DialogProps> = {
 
 export default storybookMeta;
 
+const checkDialog = async ({
+  canvasElement,
+  step,
+}: PlaywrightProps<DialogProps>) => {
+  await step("Check Dialog", async () => {
+    const canvas = within(canvasElement);
+    const buttonElement = canvas.getByText("Open dialog");
+    await userEvent.click(buttonElement);
+    await screen.findByRole("dialog");
+  });
+};
```

Comment:
Can we rename this to be more specific to what it's doing: `findDialogElement`? `checkDialog` doesn't tell me what it's checking. If we care about existence, then it's `getIsDialogVisible`.

```suggestion
const checkDialog = async ({
  canvasElement,
  step,
}: PlaywrightProps<DialogProps>) => {
  await step("Check Dialog", async () => {
    const canvas = within(canvasElement);
    const buttonElement = canvas.getByText("Open dialog");
    await userEvent.click(buttonElement);
    return await screen.findByRole("dialog");
  });
};
```

I know `async-await` does an auto-return at the end, but I prefer to be explicit so it's clear what's happening. Heck, I didn't even know about the auto-return until a few months ago since nothing else works that way in JS.

Reply:
I tried but it raised an typescript error:
```
Argument of type '() => Promise<HTMLElement>' is not assignable to parameter of type 'PlayFunction<ReactRenderer, DialogProps>'.
  Type 'Promise<HTMLElement>' is not assignable to type 'void | Promise<void>'.
    Type 'Promise<HTMLElement>' is not assignable to type 'Promise<void>'.
      Type 'HTMLElement' is not assignable to type 'void'.
```

Reply:
I think that means something is wrong with our types then right? Playwright functions don't have a return value, but ours does. I think that's why it's getting confused. But we defined this as a regular function with `PlaywrightProps`, not defining the whole function as `PlayFunction<ReactRenderer, DialogProps>` correct?

So it should be whatever return value you we want.

Reply:
I was wrong. This doesn't return anything nor do the other `step` functions. My bad.